packages/react-vapor/src/components/select/examples/ServerSideSelectExamples.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import * as _ from 'underscore';
+
+import {IReactVaporExampleState} from '../../../../docs/Reducers';
+import {withServerSideProcessing} from '../../../hoc/withServerSideProcessing/withServerSideProcessing';
+import {FilterBoxSelectors} from '../../filterBox/FilterBoxSelectors';
+import {IItemBoxProps} from '../../itemBox/ItemBox';
+import {selectWithFilter} from '../hoc/SelectWithFilter';
+import {selectWithInfiniteScroll, SelectWithInfiniteScrollProps} from '../hoc/SelectWithInfiniteScroll';
+import {ISingleSelectOwnProps, SingleSelectConnected} from '../SingleSelectConnected';
+
+const TOTAL_ENTRIES = 5000;
+const PER_PAGE = 10;
+const IMG_SIZE = 50;
+
+interface PhotoProps {
+    albumId: string;
+    id: string;
+    title: string;
+    url: string;
+    thumbnailUrl: string;
+}
+
+const clean = <T extends object>(o: T) => _(o).pick(_.identity);
+
+function usePhotosAPIMock(initialPhotos: any[] = []): [any[], (params?: any, overwrite?: boolean) => void] {

```

Comment:
you can set the default value of `overwrite` to `false`. I think it makes the code a bit clearer

Reply:
overwrite already has a default value of `true`